omnigibson/robots/__init__.py

```diff
@@ -11,3 +11,5 @@
 from omnigibson.robots.two_wheel_robot import TwoWheelRobot
 from omnigibson.robots.franka import FrankaPanda
 from omnigibson.robots.franka_allegro import FrankaAllegro
+from omnigibson.robots.franka_leap import FrankaLeap
```

Comment:
this is sorta a todo, but i wonder if we can unify all the frankas into a single class perhaps, otherwise we might possibly end up with N franka robot classes with N different grippers lol

Reply:
I think one problem with that is that basically every methods in the class will include an "if...else..." statement because these 3 robots have different joints/links lol